airbyte-integrations/connectors/source-rss/source_rss/source.py

```diff
@@ -0,0 +1,156 @@
+#
+# Copyright (c) 2022 Airbyte, Inc., all rights reserved.
+#
+
+
+from abc import ABC
+from typing import Any, Iterable, List, Mapping, MutableMapping, Optional, Tuple
+
+from datetime import datetime
+from dateutil.parser import parse
+import feedparser
+import requests
+import time
+import pytz
+from time import mktime
+
+from airbyte_cdk.sources import AbstractSource
+from airbyte_cdk.sources.streams import Stream
+from airbyte_cdk.sources.streams.http import HttpStream
+
+item_keys = [
```

Comment:
@jrhizor any scenarios where RSS feed might have other items, how will the connector handle, would it ignore or error out?

Reply:
The test feed has non-RSS-spec-standard children for `<item>`s  like `<dc:identifier>` and it ignores them.